osquery/logger/plugins/tests/aws_logger_tests.cpp

```diff
@@ -31,13 +31,20 @@ class AwsLoggerTests : public testing::Test {
   void SetUp() override {}
 };
 
+class DummyOutcome final : public Aws::Kinesis::Model::PutRecordsOutcome {
+ public:
+  bool IsSuccess() {
```

Comment:
This should be `override` right?

Reply:
The override keyword can't be used because IsSuccess is not a virtual method